File: examples/js/post__reject.js

const authToken = process.env.AIOPS_TOKEN || 'YOUR_TOKEN';
// 示例：POST /reject
fetch(`http://localhost:8000/api/v1/reject`, {
    method: 'POST',
    headers: { Authorization: `Bearer ${authToken}` }
})
.then(res => res.json())
.then(data => console.log(data))
.catch(err => console.error('Request error:', err));

File: examples/python/post__reject.py
# -*- coding: utf-8 -*-
import requests

# 示例：POST /reject
# 使用 POST 方法请求 /reject
url = "http://localhost:8000/api/v1/reject"
headers = {"Authorization": "Bearer YOUR_TOKEN"}

try:
    response = requests.post(url, headers=headers)
    print("Status:", response.status_code)
    print("Response:", response.json())
except Exception as e:
    print("Request failed:", e)
